### ISSUE TITLE
Show bbox annotations

### ISSUE THREAD

**Issue by qwert1337** [cocodataset/cocoapi#138]
I created my own coco dataset with polygons as segmentation and bounding boxes. Now I would like to draw just the bounding box into an image.
Is it possible to show the bbox with showAnns() without converting the bbox to segmentation?

**Comment by wenting-zhao** [comment]
#183 Hi!  Just in case it is useful to others, I've created a pull request to address this feature.  Let me know if there is any problem with it!

**Comment by aysark** [comment]
Please address this issue/merge PR- showing bbox anno's is kind of needed.

**Comment by tloki** [comment]
Is there a plan to add this feature?

**Comment by Vik-hub19** [comment]
**No**, I tried the same and it was **not** possible. The work around is to extract bbox info and use cv2 or other methods to draw a rectangle on the image. 

_Sample Code:_
anns = coco.loadAnns(annIds)
for i in anns:
     [x,y,w,h] = i['bbox']
     cv2.rectangle(Image, (int(x), int(y)), ((int(x+w), int(y+h)), (255,0,0), 5)
     cv2.imshow(' ',Image)
plt.show()

**Comment by talbio** [comment]
is this still open?

**Comment by hlydecker** [comment]
I'll use the workaround for now, but it would be nice to just plot bounding boxes natively.

**Comment by mariaculman18** [comment]
This should be closed. It is already available with the argument draw_bbox (use as `coco.showAnns(anns, draw_bbox=True)`)

**Comment by Keiku** [comment]
Is there a way to add label text in `coco.showAnns()`?

An example is as follows.

<img width="395" alt="image" src="https://user-images.githubusercontent.com/199433/157193288-380975de-b773-48cb-b2d6-10cd2e79abae.png">

**Comment by mariaculman18** [comment]
@Keiku you can add something like this to that function, where class is the string of the class:
`ax.text(bbox_x, bbox_y, class, fontsize=15, bbox=dict(facecolor='white', alpha=0.5))`

**Comment by Keiku** [comment]
Thank you for your reply, but I couldn't do it well, so I'll think about this issue a bit more. If you know, please tell me a good example link or GitHub.

**Comment by Keiku** [comment]
@mariaculman18 Thank you. I was able to do it with the following code.

```python
fig, ax = plt.subplots(figsize=(10, 8))
ax.imshow(image)
coco.showAnns(anns, draw_bbox=True)
for i, ann in enumerate(anns):
    ax.text(anns[i]['bbox'][0], anns[i]['bbox'][1], anns[i]['category_name'], style='italic', 
            bbox={'facecolor': 'white', 'alpha': 0.7, 'pad': 5})
```

**Comment by BassieWitkin** [comment]
> This should be closed. It is already available with the argument draw_bbox (use as `coco.showAnns(anns, draw_bbox=True)`)

when I try running this code, I get the following error:
`TypeError: showAnns() got an unexpected keyword argument 'draw_bbox'`

**Comment by Keiku** [comment]
@Bassie1 Probably not `showAnns()` but `coco.showAnns()` ?

**Comment by BassieWitkin** [comment]
@Keiku yeah, the code is `coco.showAnns`, does this have to do with versions? I just installed the pycocotools library a couple of days ago

**Comment by Keiku** [comment]
@Bassie1 it's no problem with `pycocotools-2.0.4` in my environment.

**Comment by BassieWitkin** [comment]
@Keiku I have version 2.0, should that be the issue?

**Comment by Keiku** [comment]
@Bassie1 I don't know if there is a problem with that version. Please reinstall or install another version. There is also the possibility of other careless mistakes.
